platforms/android/library/src/androidTest/java/io/element/android/wysiwyg/fakes/FakeStyleConfig.kt

```diff
@@ -1,8 +1,16 @@
 package io.element.android.wysiwyg.fakes
 
+import android.content.Context
+import android.graphics.drawable.ColorDrawable
 import io.element.android.wysiwyg.utils.StyleConfig
 
-internal val fakeStyleConfig = StyleConfig(
+internal fun createFakeStyleConfig(context: Context) = StyleConfig(
```

Comment:
Is `Context` really needed here?

Reply:
Thanks, I forgot to remove this! [`f5db458`](https://github.com/matrix-org/matrix-rich-text-editor/pull/454/commits/f5db458f1cc775f3b9b7ccb025459fdc7552f571)